.changeset/soft-spies-sneeze.md

```diff
@@ -2,4 +2,4 @@
 "@rhds/elements": minor
 ---
 
-`<rh-cta>`: Added `rh-cta-lightdom.css` to reduce layout shift before element is defined
+`<rh-cta>`: Added `rh-cta-lightdom-shim.css` as an optional file to help reduce layout shift before element is defined, and declarative shadowdom is not an option.
```

Comment:
@bennypowers Thoughts on a reword?

Reply:
"where declarative shadow DOM"...

also rename the file